gpd/CloudIndexed cloud_indexed
---
gpd/GraspConfigList grasp_configs
